addons/mail/static/tests/helpers/mock_server/models/mail_tracking_value.js

```diff
@@ -47,17 +47,17 @@ patch(MockServer.prototype, "mail/models/mail_tracking_value", {
     ) {
         let isTracked = true;
         const irField = this.models["ir.model.fields"].records.find(
-            (field) => field.model === modelName && field.name === fieldName
+            (field_id) => field_id.model === modelName && field_id.name === fieldName
         );
 
         if (!irField) {
             return;
         }
 
         const values = {
-            field: irField["id"],
-            field_desc: field["string"],
-            field_type: field["type"],
+            field_id: irField["id"],
+            field_desc: irField["string"],
+            field_type: irField["type"],
```

Comment:
Keys for removed fields are no longer necessary and should be removed.
Places where they are used in mock server need to be replaced with finding ir field for `field_id` and accessing corresponding field on that model.